internal/models/source_model.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+type Source struct {
+	// Required auto-generated PK.
+	ID uint64 `db:"id" json:"id"`
+
+	// Source ID in sources
+	SourceID string `db:"source_id" json:"source_id"`
+
+	// User-facing name. Required.
+	Name string `db:"name" json:"name"`
+
+	// Organization ID. Required.
+	OrgID string `db:"org_id" json:"org_id"`
```

Comment:
What is this? TenantID? https://github.com/RedHatInsights/sources-api-go/blob/main/model/tenant.go

If so, this needs to be uint64 primary key and we should use the correct term "tenant_id" as well. We need to be very careful here, because tenant, account/external_tenant and org_id are three different numbers of a same entity.

Also, we absolutely need to associate every single record in our database with our own account_id which is our own primary table for accounts. That one also contains org_id and account_number. Please create a column and a field for that: `account_id BIGINT NOT NULL REFERENCES accounts(id),`